spec/integration/prometheus_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative '../vcr/vcr_setup'
+require_relative '../spec_helper'
+
+module Hawkular::Prometheus::RSpec
+  HAWKULAR_BASE = 'http://localhost:8080/'
+  creds = {}
+  options = {}
+
+  describe 'Prometheus/Queries' do
+    let(:cassette_name) do |example|
+      description = example.description
+      description
+    end
+
+    around(:each) do |example|
+      record('Prometheus/Queries', credentials, cassette_name, example: example)
+    end
+
+    before(:each) do
+      @client = Hawkular::Prometheus::Client.new(HAWKULAR_BASE, creds, options)
+    end
+    it 'Should Fetch Metrics' do
```

Comment:
Missing a new line here.
Also, tests should be described as phrases in the third person, current tense. So, this is good:

```ruby
it 'fetches metrics' do
```

While those are bad:

```ruby
it 'should fetch metrics'
it 'Should Fetch Metrics'
```

The reason for that is because of the `specification` formatter for RSpec, that generates a textual description of what the test suite is tested, and in proper codebases is also a specification on what the system does.

Check http://www.betterspecs.org/#should for reference.

Reply:
This is good, but it is the same, I will use the same pattern used in the other specs files and we can refactor later in a separate PR.